config/routes.rb

```diff
@@ -28,6 +28,7 @@ def put(*_args); end
       resources :portfolio_items,       :only => [:create, :destroy, :index, :show, :update] do
         resources :provider_control_parameters, :only => [:index]
         resources :service_plans,               :only => [:index]
+        resources :icon,                        :only => [:index]
```

Comment:
This route should probably be referenced via a show action.  The `icon` endpoint only returns a single entity but calling it vai an `index` method denotes an array of one of more items would be returned.  If it is true an `array` can be returned then we should call the route `/icons` and then the index method makes sense.

Reply:
It can only return one icon, so I'll update it to show.